src/connect-proxy/worker.ts

```diff
@@ -115,13 +115,7 @@ const tunnelToDevice: nodeTunnel.Middleware = (req, cltSocket, _head, next) =>
 			cltSocket.end('HTTP/1.0 403 Forbidden\r\n\r\n'),
 		)
 		.catch((err: Error) => {
-			captureException(
-				err,
-				`unexpected error establishing tunnel to ${req.url} (${err.message})`,
-				{
-					req,
```

Comment:
You should add this manual `req` passing back so that the sentry error can have the context info available